### PR TITLE
Sre 1014/sloctl docker image

### PR DESCRIPTION
**Pull request by a-hales**

### CODE REVIEW

File: codefresh.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    git: "github"
+    stage: "clone"
+
+  install:
+    title: "Install sloctl"
+    type: "freestyle" # Run any command
+    image: "ubuntu:latest" # The image in which command will be executed
+    working_directory: "${{clone}}" # Running command where code cloned
+    commands:
+      - "ls"
```
Comment (themightymuppet) [inline codefresh.yml:25]: not a big deal but you can remove this line now

**Review by themightymuppet — APPROVED** [review]

File: codefresh.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  build:
+    title: "Building docker image"
+    type: "build"
+    working_directory: "${{clone}}"
+    arguments:
+      image_name: "nobl9/sloctl"
+      dockerfile: 'Dockerfile'
+      disable_push: true
+      tag: "${{CF_BRANCH_TAG_NORMALIZED}}"
```
Comment (ian-bartholomew) [inline codefresh.yml:40]: will that use the latest tag? like `v1.0.0`?

File: codefresh.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    stage: "build"
+
+  build:
+    title: "Building docker image"
+    type: "build"
+    working_directory: "${{clone}}"
+    arguments:
+      image_name: "nobl9/sloctl"
+      dockerfile: 'Dockerfile'
+      disable_push: true
```
Comment (ian-bartholomew) [inline codefresh.yml:39]: why not just push it here?

File: codefresh.yml (hunk 1 above)
Comment (a-hales) [reply codefresh.yml]: removed!

File: codefresh.yml (hunk 2 above)
Comment (a-hales) [reply codefresh.yml]: no it won't but i fixed this in the build step by using 

`tags: 
        - "latest"
        - "v${{VERSION}}"`

File: codefresh.yml (hunk 3 above)
Comment (a-hales) [reply codefresh.yml]: you're right. I change this to push in the build step

**Pull request merged**